### PR TITLE
Fix stylistic issues and typos in Czech localization

### PR DESCRIPTION
**Pull request by chylex**

### CODE REVIEW

**Comment by chylex** [comment]
Paging @Sophia2329 to double-check the corrections.

**Comment by Sophia2329** [comment]
At least I got it going. I'm not up to ensuring perfection, it seems.

**Comment by Sophia2329** [comment]
Glad to have you do the 2nd run of translation, I suspected it was needed but wow is it not kind on my confidence

>39

Guess it is somewhat redundant but do people really know what "našeptávač" means?

>48

Is that the official translation?

>51, 60

Inheriting ambiguities from EN version

>64, 66, 70

Source?

> Share on BetterTweetDeck

That's what's in the original *shrug*

**Comment by chylex** [comment]
> 39

Hopefully; the typeahead option is used when writing tweets, not in search.

> 48

Yes - [odznak](https://help.twitter.com/cs/rules-and-policies/twitter-rules); also used the term _lajk_ in other places which is used on the official website

> 51, 60

I suppose, yea... hopefully that will be solved with the new settings UI.

> 64, 66, 70

Official Firefox localization, do other browsers translate 'tab' differently? I suppose we can use just Chrome's translation, but I don't have it at hand right now.

> Share on BetterTweetDeck

I replaced these with the actual string used for the context menu item, which was in your translation: `"shareOnTD": { "message": "Sdílet na TweetDecku" },`.

**Comment by Sophia2329** [comment]
I remember both "tab" and "bookmark" being "záložka", so I appreciated Chrome changing "tab" to "karta", but I included both in thinking Firefox used the former. Good to know

**Comment by chylex** [comment]
Looked into it further...

* **Internet Explorer, Edge** - záložka
* **Firefox** - panel
* **Chrome** - karta
* **Opera, Vivaldi** - list

Basically, fuck all of them :P

**Comment by chylex** [comment]
Considering Chrome and FF are the most used out of the supported browsers, I'll update the translation for those two.

**Comment by Sophia2329** [comment]
Don't you think it's a good idea to run this by more people?

**Comment by chylex** [comment]
I think we can merge it, unless you see any issues.

File: src/_locales/cs/messages.json [hunk 1]
```diff
@@ -36,83 +36,83 @@
...
-  "ctrl_changes_interactions__enabled": { "message": "Podržením <kbd>Ctrl</kbd> (nebo <kbd>Cmd</kbd> na Mac) při klikání na: favorite / retweet / uložit vám umožní začít účet sledovat z"},
+  "ctrl_changes_interactions__enabled": { "message": "Podržení <kbd>Ctrl</kbd> (nebo <kbd>Cmd</kbd> na Mac) při kliknutí na: lajk / retweet / stažení vám umožní sledovat účet z"},
   "ctrl_changes_interactions__mode__owner": { "message": "účtu, jemuž patří sloupec"},
   "ctrl_changes_interactions__mode__dialog": { "message": "okénka pro vybrání sledování"},
 
   "btd_thx": { "message": "Děkujeme, že jste si nainstalovali Better TweetDeck!" },
   "btd_get_started": { "message": "Pojďme na to!" },
   "btd_whatsBTD": { "message": "Better TweetDeck byl vytvořen, aby zlepšil prohlížení <a href='https://tweetdeck.twitter.com'>tweetdeck.twitter.com</a>!" },
-  "btd_default": { "message": "Nastavení mají výchozí hodnoty, můžete si je podle sebe upravit!" },
+  "btd_default": { "message": "Nastavení mají výchozí hodnoty, můžete si je upravit podle sebe!" },
```
Comment (Sophia2329) [inline src/_locales/cs/messages.json]: Are you sure about this?

**Comment by Sophia2329** [comment]
See my PR into your fork

**Comment by Sophia2329** [comment]
We might want to translate "Compose" more literally as not to make edits later @chylex 
If you think that's fine you can mention @.eramdam and ask for the merge

**Comment by chylex** [comment]
Idk, 'Compose' is an internal term, I don't think TweetDeck ever refers to the panel that way.

**Comment by eramdam** [comment]
@Sophia2329 @chylex do you confirm that I can merge this?

**Comment by chylex** [comment]
Yea

**Pull request merged**